types/components/http/Request.d.ts

```diff
@@ -160,13 +160,13 @@ export default class Request {
      * Returns path parameters from incoming request in Object form {key: value}
      * @returns {Record<string, unknown>}
      */
-    get path_parameters(): Object;
+    get path_parameters(): Record<string, unknown>;
 
     /**
      * Returns query parameters from incoming request in Object form {key: value}
      * @returns {Record<string, unknown>}
      */
-    get query_parameters(): Object;
+    get query_parameters(): Record<string, unknown>;
```

Comment:
This record may be a bit ambiguous as we use the query-string package under the hood to parse query parameters and currently only the default options are used which ensures that only a string value is possible. But in the next update I am planning on adding the ability to configure the options passed to the query parameter parser through Server/Route options.

Reply:
I think for now we can keep the `unknown`, what do you think?

Reply:
But, if you want, I can change it to `string` and in the next update we can change it to use generics `<T>`.